Dockerfile

```diff
@@ -84,7 +85,7 @@ USER root
 # Make dir for test resources
 # Update symlnk used by Liberty to new server.  Need root.
 RUN microdnf update -y && rm -rf /var/cache/yum && \
-    microdnf install -y --nodocs vim && \
+    microdnf install -y --nodocs vim openssl && \
```

Comment:
Is `openssl` used by one of our downstream processes?

Reply:
That snuck in there with a cut and paste from some of the security/cert work I was doing in another branch. Most other images in Alvearie are adding that to their images. I was using it to see if I could use it to get the certs a different way. I can remove it from this PR if it bugs you, but it will probably come back in with the security code I am working on in another branch.